react_practice/react-apps/waitlist-page-v1/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "classnames": "^2.3.1",
```

Comment:
This allows us to compose `className` props together in a nice way (you'll see this used in `PrimaryBtn.js`)